alpine/SPECS/wazuh-agent/APKBUILD

```diff
@@ -7,7 +7,7 @@ pkgdesc="Wazuh helps you to gain security visibility into your infrastructure."
 url="https://www.wazuh.com/"
 arch="${architecture}"
 license="GPL"
-depends="bash busybox procps"
+depends="busybox procps"
```

Comment:
Why is `bash` removed as a dependency?

Reply:
The scripts work with SH and not even need bash.